File: web/assets/dart/desanitize_emails.dart
import 'dart:html';

void main() {
  querySelectorAll('a[href^="mailto:"]').forEach((emailLinkElem) {
    emailLinkElem.attributes['href'] = emailLinkElem.attributes['href']
        .replaceAll('\(at\)', '@')
        .replaceAll('\(dot\)', '.');
    emailLinkElem.text =
        emailLinkElem.text.replaceAll('\(at\)', '@').replaceAll('\(dot\)', '.');
  });
}

File: web/assets/js/desanitize_emails.js
$('a').each(function() {
    if (this.href.substring(0, 7) == 'mailto:') {
        this.href = this.href.replace(/\(at\)/g, '@').replace(/\(dot\)/g, '.');
        this.text = this.text.replace(/\(at\)/g, '@').replace(/\(dot\)/g, '.');
    }
});
